### DISCUSSION TITLE
Automatic build and flash for ESP32

### DISCUSSION THREAD

**Discussion by jomardyan** [nanoframework/Home#976] [Ideas]
Hello, are you going to create an automatic build and flash system/script for **ESP32** devices ?  

@josesimoes 

Regards

**Comment by Ellerbach** [comment]
> Hello, are you going to create an automatic build and flash system/script for **ESP32** devices ?

Hi! What do you mean exactly? There are dev contains which allow you to build the image fully if you need.

Otherwise, nanoff tool is your best friend. Just flash the device, and then debug.

Maybe you can give more information on your real scenario?

**Reply by josesimoes** [reply]
Yes please detail what you're requesting a bit so we can understand what you're trying to accomplish/suggesting.
